net/proxy_resolution/proxy_config_service_tor.cc

```diff
@@ -210,10 +212,13 @@ void ProxyConfigServiceTor::SetProxyAuthorization(
     auto* map = GetTorProxyMap(service);
     if (host_port_pair.username() == username) {
       // password is a int64_t -> std::to_string in milliseconds
-      int64_t time = strtoll(host_port_pair.password().c_str(), nullptr, 10);
+      CHECK_GE(host_port_pair.password().size(), 8u);
+      // int64_t time = strtoll(host_port_pair.password().c_str(), nullptr, 10);
```

Comment:
Was this forgotten?

Reply:
Fixed in https://github.com/brave/brave-core/pull/28471/commits/668cc8bcfc70a3beb708f788a8d3cd93e87c27e5